node/src/BaseClient.ts

```diff
@@ -1378,8 +2004,11 @@ export class BaseClient {
      * formatted as [key, value]. If no element could be popped and the timeout expired, returns Null.
      *
      * @example
-     *     await client.brpop(["list1", "list2"], 5);
-     *    ["list1", "element"]
+     * ```typescript
+     * // Example usage of brpop method to block and wait for elements from multiple lists
+     * const result = await client.brpop(["list1", "list2"], 5);
```

Comment:
```suggestion
     * const result = await client.brpop(["list1", "list2"], .5);
```
Use FP in example, but keep in mind that redis 6 does not support FP there